R/Spectra.R

```diff
@@ -331,31 +331,23 @@ NULL
 #' - `uniqueMsLevels()`: get the unique MS levels available in `object`. This
 #'   function is supposed to be more efficient than `unique(msLevel(object))`.
 #'
-#' @section Data subsetting, filtering and merging:
+#' @section Filter spectra data:
 #'
-#' Subsetting and filtering of `Spectra` objects can be performed with the below
-#' listed methods.
+#' Filter a `Spectra` object based on the spectra data. This includes subset
+#' operations that reduce the number of spectra in the object as well as filters
+#' that reduce the *content* of the `Spectra` object. See section
+#' *Filter peaks data* below for functions that filter the peaks data of a
+#' `Spectra`.
```

Comment:
maybe to make it extremely clear, we can say the number of spectra in the Spectra object will not change with filters until applyProcessing() is run ?

Reply:
I guess here or at the beginning of the filter section